app/components/Exchange/OrderBook.jsx

```diff
@@ -427,49 +424,31 @@ class OrderBook extends React.Component {
         }
     }
 
-    componentWillUpdate() {
+    getSnapshotBeforeUpdate() {
         if (!this.props.horizontal) {
```

Comment:
To make the user able to disable auto scroll, this has to be `if (!this.props.horizontal && this.state.autoScroll) {`